### PR TITLE
Proper command handling + beautifullified (lol) code

### PR DESCRIPTION
**Pull request by HimbeersaftLP**

### CODE REVIEW

**Comment by HimbeersaftLP** [comment]
Be aware: This is untested

File: bot.js [hunk 1]
```diff
@@ -10,128 +10,148 @@ var token = config.token;
...
-    } catch (err) {
-      if (err !== null && typeof err === 'object') {
-        err = util.inspect(err)
-      }
-      msg.channel.send(":x: Error!\n\`\`\`\n"+err+"\n\`\`\`").then(m => m.delete(60000));
-      // TODO: .catch handle message with over 2k chars
+        console.log(temp);
+        break;
+      case 'ping':
+        var embed = getEmbed('?? Ping!');
```
Comment (Thunder33345) [inline bot.js:66]: something gone wrong here?
it's `:ping_pong:` not "??"

Comment (Thunder33345) [reply bot.js:66]: Same below on L 71

File: bot.js [hunk 2]
```diff
@@ -10,128 +10,148 @@ var token = config.token;
...
+          emode = false;
+          msg.edit('Embed mode disabled');
+        } else if (emode == false) {
+          emode = true;
+          msg.edit('Embed mode enabled');
+        }
+        break;
+      case 'embed':
+        msg.edit("", {
+          embed: new Discord.RichEmbed().setDescription(args.join(' '))
```
Comment (Thunder33345) [inline bot.js:101]: mind changing this to getEmbed? (so it's colourful)

Comment (Thunder33345) [reply bot.js:101]: same on #L108

File: bot.js [hunk 3]
```diff
@@ -10,128 +10,148 @@ var token = config.token;
...
     process.exit(1);
   }
 });
 
 bot.on('message', msg => {
   if (msg.author.id !== bot.user.id) return;
+  if (S(message.content).startsWith('..')) {
+    var cmd = message.content.split(" ")[0];
+    cmd = S(cmd).chompLeft('..').s;
+    var args = message.content.split(" ").slice(1);
```
Comment (Thunder33345) [inline bot.js]: Seems fine but i dont know what all of these does, i guess like explode, then substring?
so i guess it's like 
```php
$msg;
$cmd = explode($msg," ");
$cmd = substr(/*remove the .. on start*/);
$args = explode($msg," ");
unset($args[first($args)])
```

**Review by Thunder33345 — CHANGES_REQUESTED** [review]

**Comment by Thunder33345** [comment]
that's fast before i even said about "might just replace .. with config.prefix"

**Comment by Thunder33345** [comment]
@HimbeersaftLP is it tested?
please dont merge without testing

**Comment by HimbeersaftLP** [comment]
WTF GIT WHY,  I didn't want to merge

**Comment by Thunder33345** [comment]
also push to develop next time, which contains untested code
we can merge onto master once a feature set is completed 
feel free to create more branches for X specific feature
but they should only be merged into develop(unless hotfixes with high priority)

**Comment by Thunder33345** [comment]
/shrug
screw workflow i guess
says the git n00b

**Pull request merged**